core/src/main/java/com/netease/arctic/table/BaseUnkeyedTable.java

```diff
@@ -139,7 +143,7 @@ public Map<Integer, SortOrder> sortOrders() {
   public Map<String, String> properties() {
     return icebergTable.properties().entrySet()
         .stream()
-        .filter(e -> !TableConstants.HIDDEN_PROPERTIES.contains(e.getKey()))
+        .filter(e -> !TableProperties.PROTECTED_PROPERTIES.contains(e.getKey()))
         .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
   }
 
```

Comment:
for keyedHiveTable,  the base table implement is UnkeyedHiveTable.  this may cause keyed hive table can't read protected properties

Reply:
yes, so i override `properties` in base table impelementation